tensorflow_datasets/scripts/random_image_generator.py

```diff
@@ -0,0 +1,102 @@
+import os
```

Comment:
Could you run pylint to format the file. You also need to add a top level docstring to explain what this file does.

Reply:
I have already format the file with pylint. Should i need to add the license and the copyright comment as well.

Reply:
Could you use https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style ?

Reply:
No need to add the license. It will be added automatically. But thank you for asking.